openshift/templates/compliance-backend.yml

```diff
@@ -96,15 +98,17 @@ objects:
             value: "${POSTGRESQL_SHARED_BUFFERS}"
           - name: RAILS_ENV
             value: "${RAILS_ENV}"
+          - name: PATH_PREFIX
+            value: "${PATH_PREFIX}"
           livenessProbe:
             httpGet:
-              path: /r/insights/platform/compliance/api-docs/index.html
+              path: /api/compliance/api-docs/index.html
```

Comment:
Couldn't this also be using the env variables?

Reply:
I'm not sure how, these YAML files are just uploaded to Openshift.